openshift/echo-api-template.yml

```diff
@@ -0,0 +1,99 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: echo-api
+  annotations:
+    description: "3scale Echo API"
+    iconClass: "icon-ruby"
+    tags: "api,echo-api,3scale"
+
+objects:
+
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: echo-api
+  imagePullSecrets:
+    - name: quay-auth 
+
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    name: echo-api-production
+  spec:
+    replicas: 1
+    selector:
+      deploymentconfig: echo-api-production
+    strategy:
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 180
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          deploymentconfig: echo-api-production
+      spec:
+        containers:
+        - image: ${ECHOAPI_IMAGE}
+          imagePullPolicy: Always
+          name: echo-api-production
+          livenessProbe:
```

Comment:
Make the liveness probe just check the socket. doesn't make sense to check the same thing two times.